capstone/fitness/src/main/webapp/profile-info.html

```diff
@@ -41,6 +41,13 @@
         <br>
         <br>
 
+        <div class="input-container">
+          <label for="mileTime">How fast can you run you a mile in minutes (it's okay to estimate)</label>
+          <input class="separator" type="number" name="mileTime" min="1" step="0.01" required>
```

Comment:
It's a little confusing that mileTime definition is different here compared to in the servlet where you're dividing by 60. Maybe you can rename this to minPerMile?

Reply:
Well I was naming it trying to hint that it is the time in which the user can run a mile, and in data store everything is stored in hours, but asking the user for their time in hours for 1 mile seemed a bit strange.